File: Python/Program/Product.py
class Product:
    # Private Attributes
    __idProduct = ""
    __name = ""
    __brand = ""
    __price = ""
    
    # Constructor
    def __init__(self, idProduct="", name="", brand="", price=""):
        self.__idProduct = idProduct
        self.__name = name
        self.__brand = brand
        self.__price = price

    # Getter and Setter
    # Getter 
    def getIdProduct(self):
        return self.__idProduct

    def getName(self):
        return self.__name

    def getBrand(self):
        return self.__brand

    def getPrice(self):
        return self.__price

    # Setter 
    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    def setName(self, name):
        self.__name = name

    def setBrand(self, brand):
        self.__brand = brand

    def setPrice(self, price):
        self.__price = price

File: PHP/Program/Product.php
<?php

class Product {
    // Private Attributes
    private $idProduct = "";
    private $name = "";
    private $brand = "";
    private $price = "";
    
    // Constructor
    public function __construct($idProduct="", $name="", $brand="", $price="") {
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }
    
    /*Getter & Setter*/
    // Getter 
    public function getIdProduct() {
        return $this->idProduct;
    }

    public function getName() {
        return $this->name;
    }

    public function getBrand() {
        return $this->brand;
    }

    public function getPrice() {
        return $this->price;
    }

    // Setter 
    public function setIdProduct($idProduct) {
        $this->idProduct = $idProduct;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function setBrand($brand) {
        $this->brand = $brand;
    }

    public function setPrice($price) {
        $this->price = $price;
    }
}

?>
